specification/nodes/_types/Stats.ts

```diff
@@ -65,22 +142,52 @@ export class KeyedProcessor {
 }
 
 export class AdaptiveSelection {
-  avg_queue_size: long
-  avg_response_time: long
+  avg_queue_size?: long
+  avg_response_time?: long
   avg_response_time_ns: long
```

Comment:
This field (and some others in other classes) is left required. Is this intentional?